themes/base.theme.bash

```diff
@@ -399,8 +402,21 @@ function nvm_version_prompt() {
 	fi
 }
 
+function node_native_version_prompt() {
+	local node
+	if which -s node; then
```

Comment:
you can use `command_exists`, and probably should log here as well

Reply:
done :) -- feel free to massage as you see fit before merging